src/flags/sorting.rs

```diff
@@ -516,50 +493,48 @@ mod test_dir_grouping {
         assert_eq!(None, DirGrouping::from_config(&Config::with_none()));
     }
 
-    #[test]
-    fn test_from_config_empty() {
-        let yaml_string = "---";
-        let yaml = YamlLoader::load_from_str(yaml_string).unwrap()[0].clone();
-        assert_eq!(None, DirGrouping::from_config(&Config::with_yaml(yaml)));
-    }
-
     #[test]
     fn test_from_config_first() {
-        let yaml_string = "sorting:\n  dir-grouping: first";
-        let yaml = YamlLoader::load_from_str(yaml_string).unwrap()[0].clone();
-        assert_eq!(
-            Some(DirGrouping::First),
-            DirGrouping::from_config(&Config::with_yaml(yaml))
-        );
+        let mut c = Config::with_none();
+        c.sorting = Some(Sorting {
+            column: None,
+            reverse: None,
+            dir_grouping: Some(DirGrouping::First),
+        });
+        assert_eq!(Some(DirGrouping::First), DirGrouping::from_config(&c));
     }
 
     #[test]
     fn test_from_config_last() {
-        let yaml_string = "sorting:\n  dir-grouping: last";
-        let yaml = YamlLoader::load_from_str(yaml_string).unwrap()[0].clone();
-        assert_eq!(
-            Some(DirGrouping::Last),
-            DirGrouping::from_config(&Config::with_yaml(yaml))
-        );
+        let mut c = Config::with_none();
+        c.sorting = Some(Sorting {
+            column: None,
+            reverse: None,
+            dir_grouping: Some(DirGrouping::Last),
+        });
+        assert_eq!(Some(DirGrouping::Last), DirGrouping::from_config(&c));
     }
 
     #[test]
     fn test_from_config_explicit_none() {
```

Comment:
Change the test name back to `test_from_config_empty`. It is more consistent and essentially testing the same thing as the previous implementation.